CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [0.18.4](https://github.com/eclipse/lemminx/milestone/26?closed=1) (February 14, 2022)
```

Comment:
Ah sorry, just noticed this. should be for vscode-xml. Milestone number is correct.

Reply:
Given the enhancement, it should probably be 0.19.0

Reply:
Fair enough. Let's make it 0.19.0 :)